.github/workflows/ci.yml

```
@@ -9,16 +13,19 @@ jobs:
   Test:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-latest]
+        node_arch:
+          - x64
```

This should allow e.g. the macOS matrix job to proceed if the Ubuntu job fails, for whatever reason.

```suggestion
          - x64
      fail-fast: false
```

(I hope I got the indent level right...)